adapters/cryptopunks2/index.js

```diff
@@ -0,0 +1,123 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+
+class CRYPTOPUNKS2 {
+
+    constructor() {
+        this.name = "cryptopunks2";
+        this.symbol = "PUNKS2";
+        this.token = "0xc02d332AbC7f9E755e2b1EB56f6aE21A7Da4B7AD";
+        this.protocol = "polygon";
+        this.block = 20346370;
+        this.contract = "0xc02d332AbC7f9E755e2b1EB56f6aE21A7Da4B7AD";
+        this.events = ["Transfer"],
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
+        this.chunkSize = 6;
+        this.sdk = new Ethereum(this);
+    }
+
+    run = async () => {
+        const s = await this.getSymbol();
+        this.sdk = this.loadSdk();
+        this.symbol = s;
+        await this.sdk.run();
+    };
+
+    loadSdk = () => {
+        return new Ethereum(this);
+    };
+    getSymbol = async () => {
```

Comment:
Please use `getSymbol` function like in the example.

Reply:
Your API returns an empty symbol when querying for our token contract address:

token address: 0xc02d332AbC7f9E755e2b1EB56f6aE21A7Da4B7AD
protocol: polygon

Response:
{"symbol":"","decimals":0,"currency":"usd"}

Reply:
Hi @liutaurasjanuskas 

Your API returns an empty symbol when querying for our token contract address:

token address: 0xc02d332AbC7f9E755e2b1EB56f6aE21A7Da4B7AD
protocol: polygon

Response:
{"symbol":"","decimals":0,"currency":"usd"}

Reply:
0x8250E5E61f3c35c698007DB3266f5015C34eC904

Reply:
KFMWWHYDUD8BWZ8UQ7TFYSGHZZ5GX1G3D9

Reply:
KFMWWHYDUD8BWZ8UQ7TFYSGHZZ5GX1G3D9

Reply:
getSymbol

Reply:
`token` property defines payment token, not the NFT. Make sure that you provide payment token to `getSymbol` function and it will return correct data.

Reply:
Have updated our fork with this fix... shall we do a new pull request?